browser/js/films/films.html

```diff
@@ -11,8 +9,30 @@
   </ul>
 </div>
 
-<div ng-repeat="film in films">
-	<a ng-href="/products/{{film._id}}">{{film.title}}</a>	
-  <p ng-repeat="category in film.categories">{{category.name}}</p>
+<!-- <div ng-repeat="film in films"> -->
+	<!-- <a ng-href="/products/{{film._id}}">{{film.title}}</a>	 -->
+  <!-- <p ng-repeat="category in film.categories">{{category.name}}</p> -->
   <!--   <p>{{film.categories.name}}</p> -->	
-</div>
\ No newline at end of file
+<!-- </div> -->
+
```

Comment:
The name *browserfilm* is a bit confusing to me, maybe we could name it something like *filmcard*?